debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -405,14 +422,16 @@ public static BinaryHandlingMode parse(String value, String defaultValue) {
     public static final Field SNAPSHOT_MAX_THREADS = Field.create("snapshot.max.threads")
             .withDisplayName("Snapshot maximum threads")
             .withType(Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_SNAPSHOT, 7))
             .withWidth(Width.SHORT)
             .withImportance(Importance.MEDIUM)
             .withDefault(1)
             .withValidation(Field::isPositiveInteger)
             .withDescription("The maximum number of threads used to perform the snapshot.  Defaults to 1.");
 
-    public static final Field SIGNAL_DATA_COLLECTION = Field.create("signal.data.collection")
+    public static final Field SIGNAL_DATA_COLLECTION = Field.createInternal("signal.data.collection")
```

Comment:
WDYT about this?

Reply:
Why did it change to internal?

Reply:
It's not yet really used and documented, yet. I thought it should not show up in any list of properties (for example also when requesting available parameters over the KC REST API) and that's what we do then? But I might be wrong.